go/config/decoding.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"fmt"
+	"math/big"
+	"reflect"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// bigIntHookFunc is a mapstructure HookFunc to handle both ints and strings for big.Int
+func bigIntHookFunc() mapstructure.DecodeHookFuncType {
```

Comment:
Comment why it's needed. Guessing viper has no clue about big Ints, but would be good to have elaboration here